lib/layers/awsUtils/dynamodb.js

```diff
@@ -36,6 +36,37 @@ async function getOne(pk, sk) {
   return null;
 }
 
+/**
+ * Increments a counter value in DynamoDB; creates item counter if it doesn't exist.
```

Comment:
This is not true.  ADD requires the item to exist first.

Reply:
Sorry - jumped the line on this one. Turns out my approach is an older way of addressing the issue. SDK automagically sets to 0 and then increments based on the ADD #.  Good to go! 👍🏻